.github/workflows/generate-user-stories.yml

```diff
@@ -17,11 +17,11 @@ jobs:
           echo -e '---\nlayout: page-with-side-nav\ntitle: User Stories in Ontwikkeling\n---\n' >> user-stories-dev.md
           echo -e '# User stories\n' >> user-stories-dev.md
 
-      - name: List 'v1.5' user stories
+      - name: List 'v2.0' user stories
```

Comment:
Ik heb eerlijk gezegd geen idee of dit een 2.0 gaat worden (met breaking changes tov 1.5).

Reply:
Indien dit niet het geval is, is dat zo weer aangepast. Ik laat het even zo staan zodat het ons i.i.g. triggert.

Reply:
Aangepast